TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -533,6 +533,14 @@ public override bool Visit(TruncateStatement truncateStatement)
 
                 return true;
             }
+            public override bool Visit(LockTableStatement lockTableStatement)
+            {
+                _writer.WriteLine($"line{lockTableStatement.Line}: {nameof(lockTableStatement)}");
```

Comment:
Please add a space between 'line' and the actual number, correct the nameof expression. We want the name of the type not the parameter. Check other methods as a guide.